From 3f856c2714413f5917c1253b5a8c055f47a14c91 Mon Sep 17 00:00:00 2001
From: Zach Jang <zachjang@google.com>
Date: Thu, 17 Mar 2016 17:11:47 -0700
Subject: [PATCH 23/55] Update Security String to 2016-04-02 in preparation for
 April 2016 Security OTA v2 - kernel updates with patches for CVE-2015-1805

b/27724967

Change-Id: Ie9daf35dcba3f844c925358daf2646b7ab5b5f5f

Fix merge conflict to mnc-mr2-release w/ security string change to 2016-04-02
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 8cc6900..40ddfa0 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -103,7 +103,7 @@ ifeq "" "$(PLATFORM_SECURITY_PATCH)"
   # Can be an arbitrary string, but must be a single word.
   #
   # If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-  PLATFORM_SECURITY_PATCH := 2016-04-01
+  PLATFORM_SECURITY_PATCH := 2016-04-02
 endif
 
 ifeq "" "$(PLATFORM_BASE_OS)"
-- 
2.7.4

